src/main/java/com/endava/soa_5/basic/Exercise_4.java

```diff
@@ -0,0 +1,27 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_4 {
+    //Write 3 methods that calculates sum of numbers from 0 to a given number
+    int result;
```

Comment:
be aware of class variables, in this case if will be called 2 or more methods, the same variable will be used, and the second method will increase the same variable (so the second one will not start from zero)
Better to use local variable in each method.

Reply:
Done!